### PR TITLE
Tribute page for review

### PR DESCRIPTION
**Pull request by Stanleeeeee**
Tribute page, microverse admission project. This will be reviewed by me

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Tribute Page</title>
+    <link rel="stylesheet" href="./style.css">
+</head>
+<body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    
+    <main id="main">
+      <h2 id="title">Tribute to Mrs Elizabeth AWando</h2>
```
Comment (Stanleeeeee) [inline index.html:13]: It would have been better to section your page. It makes your code look professional

File: README.md [hunk 2]
```diff
@@ -1,73 +1 @@
...
-## Acknowledgments
-
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
-
-## 📝 License
-
-This project is [MIT](./MIT.md) licensed.
+Tribute_page
```
Comment (Stanleeeeee) [inline README.md:1]: Please use the microverse standardized format for writing your Readme file. It explains your work and makes it Stand out. Refer to the microverse readme template to make a good readme file

**Review by Stanleeeeee — COMMENTED** [review]
download
Nice Work! You are almost there.
There is no linters file in this your work. Create a linters workflow for this  project

# Changes required :recycle: 

Please check my comments above to make the required changes.

Submit a re-review when you are done with the corrections

**Review by aimemalaika — APPROVED** [review]
Hi @Stanleeeeee, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

133591850-fdcb7dd3-dd7a-4274-81e7-0e59434148f7

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**